app/objects/c_report.py

```diff
@@ -29,13 +32,15 @@ class VulnerabilityReport(FirstClassObjectInterface, BaseObject):
     def unique(self):
         return self.hash('%s' % self.id)
 
-    def __init__(self, id=None, name=None, hosts=None, scope=None, **kwargs):
+    def __init__(self, id=None, name=None, hosts=None, scope=None, nodes=[], edges=[], **kwargs):
         super().__init__()
         self.id = id or str(uuid.uuid4())
         self.name = name if name else 'vulnerability-report-%s' % date.today().strftime("%b-%d-%Y")
         self.hosts = hosts or dict()
         self.scope = scope
-        self.network_map = None
+        self.network_map = nx.Graph()
```

Comment:
No issues with the graph viz at all? It an just use networkx object with no issue, same as the previous dict structure?

Reply:
Technically the current implementation for creating our viz structure in pathfinder_gui.py iterates through the structure (be it NX or pydict), so the end result is the same. I didn't change how the NX graph gets passed to the frontend cause that seems like a whole thing and not in scope of this ticket.